microsoft-edge/devtools-guide-chromium/resources/index.md

```diff
@@ -20,139 +20,173 @@ ms.date: 05/04/2021
+
+In various tools that list resource files for the webpage, to inspect a resource file in the **Network** tool, right-click the resource file and then select **Reveal in Network panel**.  You may need to first refresh the webpage while the **Network** tool is open.
+
+For example, to go from the **Page** tab's resource list in the **Sources** tool to the **Network** tool:
+
+1. In a new window or tab, go to [https://microsoftedge.github.io/Demos/demo-to-do/](https://microsoftedge.github.io/Demos/demo-to-do/).
+
+1. To open DevTools, right-click the webpage, and then select **Inspect**.  Or, press `Ctrl`+`Shift`+`I` (Windows, Linux) or `Command`+`Option`+`I` (macOS).  DevTools opens.
 
-<!-- to do x, -->
+1. In DevTools, on the main toolbar, select the **Network** tab.  If that tab isn't visible, click the **More tabs** (![the More tabs button](../media/more-tabs-icon-light-theme.png)) button, or else the **More Tools** (![More Tools icon](../media/more-tools-icon-light-theme.png)) button.
 
-1. Open [Log network activity](../network/index.md#log-network-activity) (in the article  _Inspect network activity_) in a new tab or window.
+1. Press `Ctrl`+`R` to refresh the page.  The received resource files that make up the webpage are now added as rows in the **Network** tab.  The **demo-to-do/** folder row represents the HTML page - in this case, [index.html](https://github.com/MicrosoftEdge/Demos/tree/main/demo-to-do).
 
-<!-- 1. Do the steps in that section, and then continue below. -->
+1. In DevTools, click the **Sources** tab.
 
- <!-- open [link] in a new tab or window, and then continue below.. -->
-<!--why?-->
+1. In the **Page** tab in the upper left, right-click a resource file, and then select **Reveal in Network panel**:
 
-<!-- need lead-in -->
+   ![The 'Reveal in Network panel' right-click menuitem on a resource file in the Page tab of the Sources tool](../media/resources-sources-page-reveal-in-network-panel.msft.png)
 
-![Page resources in the Network Log.](../media/resources-network-resources.msft.png)
+   If the **Reveal in Network panel** menuitem isn't listed, go to the **Network** tool and then refresh the page.
 
 
 <!-- ====================================================================== -->
-## Browse resources by directory in the Sources tool
 
-To view the resources of a webpage organized by directory:
 
-1. Open DevTools.
 
-1. Select the **Sources** tool, and then in the **Navigator** pane in the upper left, select the **Page** tab.
 
-1. Click the **More options** (...) button to the right of the **Page** tab, and then select **Group by folder**.
 
-   ![The Page tab in the Navigator pane of the Sources tool.](../media/resources-sources-page-empty.msft.png)
+<!-- ====================================================================== -->
+## Browse resource files by folder in the Page tab of the Sources tool
+
+You can use the **Sources** tool to view the webpage's resource files organized by directory, as follows:
+
```

Comment:
```suggestion

1. In a new window or tab, go to a webpage, such as the [Accessibility-testing features demo](https://microsoftedge.github.io/Demos/devtools-a11y-testing/).  (The source files for that webpage are at [MicrosoftEdge / Demos > devtools-a11y-testing](https://github.com/MicrosoftEdge/Demos/tree/main/devtools-a11y-testing).)

```

Reply:
@dan-wesley  - h2 "Browse resource files by folder in the Page tab of the Sources tool" - did you mean to apply this Suggested change?  Comment was resolved w/o changing.